README.md

```diff
@@ -1,2 +1,15 @@
-# how-to-create-react-accumulation-charts-of-syncfusion
-A quick-start react project that depicts the creation of Syncfusion Accumulation Charts with appropriate title, legend, data labels, and tooltip.
+# How to Create React Accumulation Charts of Syncfusion
+
+A quick-start project that allows you to create an accumulation charts of Syncfusion in the react application. This project also contains a code snippet to add a title, legend, data label, and tooltip to the pie chart.
+
+Documentation: https://ej2.syncfusion.com/react/documentation/accumulation-chart/getting-started/
+
+Online examples: https://ej2.syncfusion.com/react/demos/#/material/chart/default-pie
+
+## Project prerequisites
+
+Make sure that you have the latest versions of NodeJS and visual studio code in your machine before starting to work on this project.
```

Comment:
Visual Studio Code --> Follow this casing

Reply:
Done the changes